.changeset/lazy-terms-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-linguist': minor
```

Comment:
This can be a patch as the plugin is still on a `0.x.y` version & it is not a breaking change.

```suggestion
'@backstage/plugin-linguist': patch
```